documentation/1.9/content/userguide/tools/update.md

```diff
@@ -19,7 +19,7 @@ In WLST online mode, simply add the information on how to connect to the WebLogi
 
     $ weblogic-deploy\bin\updateDomain.cmd -oracle_home c:\wls12213 -domain_type WLS -domain_home domains\DemoDomain -archive_file DemoDomain.zip -model_file DemoDomain.yaml -variable_file DemoDomain.properties -admin_url t3://127.0.0.1:7001 -admin_user weblogic
 
-As usual, the tool will prompt for the password (it can also be supplied by piping it to standard input of the tool).
+As usual, the tool will prompt for the password (it can also be supplied by piping it to standard input of the tool). To bypass the prompt, you can use one of two options. Store the password in an environment variable, and use the variable name with command line option -admin_pass_env. Store the password as the only variable in a file. Provide the file name with command line option -admin_pass_file.
```

Comment:
file name -> filename 
and same comment for "only variable in a file"

Reply:
file name (adj) (n)
Use two words in instructional text, but use filename as a variable in code instructions and commands.